diff --git a/node_modules/tsvesync/dist/lib/fans/humidifier.js b/node_modules/tsvesync/dist/lib/fans/humidifier.js
index 2624ac7..bdfe1f8 100644
--- a/node_modules/tsvesync/dist/lib/fans/humidifier.js
+++ b/node_modules/tsvesync/dist/lib/fans/humidifier.js
@@ -157,7 +157,7 @@ class VeSyncHumidifier extends vesyncFan_1.VeSyncFan {
             throw new Error(error);
         }
         logger_1.logger.debug(`Setting mode to ${mode} for device: ${this.deviceName}`);
-        const [response, status] = await this.callApi('/cloud/v2/deviceManaged/bypassV2', 'post', {
+        const [response, status] = await this.callApi('/cloud/v2/deviceManaged/bypassV2', 'put', {
             ...helpers_1.Helpers.reqBody(this.manager, 'bypassV2'),
             cid: this.cid,
             configModule: this.configModule,
@@ -193,7 +193,7 @@ class VeSyncHumidifier extends vesyncFan_1.VeSyncFan {
             throw new Error(error);
         }
         logger_1.logger.debug(`Setting target humidity to ${humidity}% for device: ${this.deviceName}`);
-        const [response, status] = await this.callApi('/cloud/v2/deviceManaged/bypassV2', 'post', {
+        const [response, status] = await this.callApi('/cloud/v2/deviceManaged/bypassV2', 'put', {
             ...helpers_1.Helpers.reqBody(this.manager, 'bypassV2'),
             cid: this.cid,
             configModule: this.configModule,
diff --git a/node_modules/tsvesync/src/lib/fans/humidifier.ts b/node_modules/tsvesync/src/lib/fans/humidifier.ts
index 69a08ba..5bbaf33 100644
--- a/node_modules/tsvesync/src/lib/fans/humidifier.ts
+++ b/node_modules/tsvesync/src/lib/fans/humidifier.ts
@@ -186,7 +186,7 @@ export class VeSyncHumidifier extends VeSyncFan {
         logger.debug(`Setting mode to ${mode} for device: ${this.deviceName}`);
         const [response, status] = await this.callApi(
             '/cloud/v2/deviceManaged/bypassV2',
-            'post',
+            'put',
             {
                 ...Helpers.reqBody(this.manager, 'bypassV2'),
                 cid: this.cid,
@@ -230,7 +230,7 @@ export class VeSyncHumidifier extends VeSyncFan {
         logger.debug(`Setting target humidity to ${humidity}% for device: ${this.deviceName}`);
         const [response, status] = await this.callApi(
             '/cloud/v2/deviceManaged/bypassV2',
-            'post',
+            'put',
             {
                 ...Helpers.reqBody(this.manager, 'bypassV2'),
                 cid: this.cid,
